helm/nessie/templates/configmap.yaml

```diff
@@ -129,28 +129,28 @@ data:
 
     {{- if .Values.catalog.enabled -}}
     {{- list .Values.catalog.iceberg $map | include "nessie.applyCatalogIcebergOptions" -}}
-    {{ list .Values.catalog.storage.s3 "nessie.catalog.service.s3." $map | include "nessie.applyCatalogStorageS3RootOptions"}}
-    {{ list .Values.catalog.storage.s3.defaultOptions "nessie.catalog.service.s3.default-options." $map | include "nessie.applyCatalogStorageS3BucketOptions"}}
+    {{- list .Values.catalog.storage.s3 "nessie.catalog.service.s3." $map | include "nessie.applyCatalogStorageS3RootOptions" }}
```

Comment:
What's the difference between these?
```
{{- list ... -}}
```
and
```
{{- list ... }}
```
and
```
{{ list ... }}
```

Reply:
whitespace control before and after the braces.

https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace